chrysom/basicClient.go

```diff
@@ -111,13 +110,13 @@ type Items []model.Item
 // NewBasicClient creates a new BasicClient that can be used to
 // make requests to Argus.
 func NewBasicClient(config BasicClientConfig,
-	getLogger func(context.Context) log.Logger) (*BasicClient, error) {
+	getLogger sallust.GetLoggerFunc) (*BasicClient, error) {
 	err := validateBasicConfig(&config)
 	if err != nil {
 		return nil, err
 	}
 	if getLogger == nil {
```

Comment:
Avoid checking for `nil` here, as there should always be a logger function provided by `sallust` or another provider.